Assignment1/pages/game.vue

```diff
@@ -0,0 +1,69 @@
+<!-- <template>
+    <v-card>
+      <v-card-title>Hello Welcome to Mystery Number Madness</v-card-title>
```

Comment:
I dont see this on your page?

Reply:
Thanks for catching this! It looks like I updated my old commented out code but not my new code.